test/integration/client-navigation/test/rendering.js

```diff
@@ -358,11 +358,11 @@ export default function(render, fetch) {
         expect($('#aspath').text()).toBe('/url-prop')
       })
 
-      it('should override props.url, even when getInitialProps returns url as property', async () => {
-        const $ = await get$('/url-prop-override')
-        expect($('#pathname').text()).toBe('/url-prop-override')
-        expect($('#query').text()).toBe('0')
-        expect($('#aspath').text()).toBe('/url-prop-override')
+      it('should show error when url prop is passed', async () => {
```

Comment:
This seems breaking. Maybe we should keep override behavior?

Reply:
Yep indeed!

Reply:
Changed back to only overriding with a warning